vm/src/frame.rs

```diff
@@ -815,30 +813,39 @@ impl Frame {
         module: &str,
         symbol: &Option<String>,
     ) -> FrameResult {
-        let current_path = {
-            let mut source_pathbuf = PathBuf::from(&self.code.source_path);
-            source_pathbuf.pop();
-            source_pathbuf
+        let import = vm.builtins.get_item("__import__");
+        let module = match import {
+            Some(func) => vm.invoke(func, vec![vm.ctx.new_str(module.to_string())])?,
+            None => panic!("No __import__ in builtins"),
         };
 
-        let obj = import(vm, current_path, module, symbol)?;
```

Comment:
I think it is better to keep the import function which accepts a module as a `&str` than to fetch the `__import__` function from the builtins and invoke that one. Do we expect the `__import__` function to be overridden by python code?

Reply:
The whole point in me adding this is to override `__import__`. You can do for example:
```py
import builtins
>>> def f(name, globals=None, locals=None, fromlist=(), level=0):
...     print(name)
... 
>>> builtins.__import__ = f
>>> import a
a
>>> import requests
requests
```
You can read about it [here](https://docs.python.org/3/library/functions.html#__import__).

I will open an issue with my plan and maybe we can do it in some other way.

Reply:
I opened #659 that I want the __import__ for. The point is to override the function on WASM.

Reply:
Oh, is this possible, I did not know this. In that case, good change! Is this a good thing of python?

Reply:
One example use is for packaged applications to be able to `import` stuff from other packaged files.
I think there are now more layers of higher and lower level than just `__import__` in `importlib`, but this will probably be enough for now.

Reply:
> Oh, is this possible, I did not know this. In that case, good change! Is this a good thing of python?

Not sure if this is a good thing but it is probably the easiest way to change import functionality.